ledger-tool/src/main.rs

```diff
@@ -1069,6 +1072,13 @@ fn main() {
                         .value_name("FILENAME")
                         .help("Record slots to a file"),
                 )
+                .arg(
+                    Arg::with_name("record_txs")
```

Comment:
The mechanism to move data is completely different so there is no conflict with anything from https://github.com/solana-labs/solana/pull/34246 right ?

Reply:
There is no conflict, the functionality is separate.